examples/0_Simple/vectorAdd/wrapper.sh

```diff
@@ -1,44 +0,0 @@
-#!/bin/sh -e
-
-# PATH of the example to run. Should be e.g. examples/0_Simple/vectorAdd/
```

Comment:
Keeping a shell script doing things manually rather than with gpr was nice to keep in mind the different steps involved in building a cuda program "without magic". I guess it's a pain to keep up to date though.